Code/Framework/AzToolsFramework/AzToolsFramework/AssetEditor/AssetEditorWidget.cpp

```diff
@@ -210,8 +211,9 @@ namespace AzToolsFramework
             else
             {
                 m_adapter = AZStd::make_shared<AZ::DocumentPropertyEditor::ReflectionAdapter>();
-                m_dpe = new DocumentPropertyEditor(this);
-                propertyEditor = m_dpe;
+                m_filteredWidget = new FilteredDPE(this);
+                m_dpe = m_filteredWidget->GetDPE();
+                propertyEditor = m_filteredWidget;
```

Comment:
Is there a reason why the FilteredDPE can't just be a DPE?

Reply:
do you mean, why don't I derive FilteredDPE from DocumentPropertyEditor? DocumentPropertyEditor is a QScrollArea (so it can scroll its contents if necessary, while FilteredDPE has to hold a DPE but keep its search box outside of the scroll area, so it doesn't scroll. That's why it's a "hasA" rather than an "isA"